examples/with-jest-babel/pages/index.js

```diff
@@ -0,0 +1,48 @@
+import Head from 'next/head'
+import styles from './index.module.css'
+
+const Home = () => (
+  <div className={styles.container}>
+    <Head>
+      <title>Create Next App</title>
+      <link rel="icon" href="/favicon.ico" />
+    </Head>
+
+    <main>
+      <h1 className={styles.title}>
+        Welcome to <a href="https://nextjs.org">Next.js!</a>
+      </h1>
+
+      <p className={styles.description}>
+        Get started by editing <code>pages/index.js</code>
+      </p>
+
+      <div className={styles.grid}>
+        <a href="https://nextjs.org/docs" className={styles.card}>
+          <h3>Documentation &rarr;</h3>
+          <p>Find in-depth information about Next.js features and API.</p>
+        </a>
+
+        <a href="https://nextjs.org/learn" className={styles.card}>
+          <h3>Learn &rarr;</h3>
+          <p>Learn about Next.js in an interactive course with quizzes!</p>
+        </a>
+
+        <a
+          href="https://github.com/vercel/next.js/tree/master/examples"
+          className={styles.card}
+        >
+          <h3>Examples &rarr;</h3>
+          <p>Discover and deploy boilerplate example Next.js projects.</p>
+        </a>
+
+        <a href="https://vercel.com/new" className={styles.card}>
+          <h3>Deploy &rarr;</h3>
+          <p>Instantly deploy your Next.js site to a public URL with Vercel.</p>
+        </a>
+      </div>
```

Comment:
what's the reason the example doesn't show using [`Image` like in the default CNA template](https://github.com/vercel/next.js/blob/canary/packages/create-next-app/templates/default/pages/index.js#L63)? (although it *does* include the `public/zeit.svg` image)

Reply:
Hey @stefanprobst, at one point the [footer was removed](https://github.com/vercel/next.js/commit/483bd3dddac631386c600230c822abb7e5140386#diff-62b427c811d505c5f02532df183c2778537287ca6c0d265dfa403c6555628f98L48). But good catch, I'll add it to the example.